tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/ProjectVersion.java

```diff
@@ -0,0 +1,24 @@
+package org.hawkular.apm.tests;
```

Comment:
Could you move these test classes into a sub-package - its just that all tests for APM belong to child packages of org.hawkular.apm.tests - so having classes in this package would be ok if they were applicable to all tests, but they are only going to be used when testing the instrumentation rules, so I think they should be put in a more specific package? Let me know if not clear or if you think they are generic enough.